packages/web/src/pages/sign-up-page/pages/CreateEmailPage.tsx

```diff
@@ -19,6 +19,7 @@ import { useDispatch, useSelector } from 'react-redux'
 import { Link } from 'react-router-dom'
 import { toFormikValidationSchema } from 'zod-formik-adapter'
 
+import { audiusQueryContext } from 'app/AudiusQueryProvider'
```

Comment:
+1 to doing this in mobile as well so we dont have to useContext, so it can be defined once outside the component